### DISCUSSION TITLE
How to set image label "nerdctl/platform" when commit a container?

### DISCUSSION THREAD

**Discussion by huanghantao** [containerd/nerdctl#1052] [Q&A] [answered]
When I commit a container, it reports:

```bash
WARN[0000] Image lacks label "nerdctl/platform", assuming the platform to be "linux/amd64"
```

However, I don't found a flag to set label.

**Comment by junnplus** [comment]
Please provide a reproducible way, and what is your environment, VM or host?

**Reply by huanghantao** [reply]
The command I executed:

```bash
nerdctl --namespace k8s.io commit 21761c9464c1 test:v0.0.1
WARN[0000] Image lacks label "nerdctl/platform", assuming the platform to be "linux/amd64"
sha256:ccee20a58981bbab0b4ded4a143b2e1e3c53546bbc5dfe06751dac072b5b61d8
```

environment is virtualbox.

other:

```bash
uname -a
Linux k8s-master-node 3.10.0-1160.el7.x86_64 #1 SMP Mon Oct 19 16:18:59 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

cat /etc/redhat-release
CentOS Linux release 7.9.2009 (Core)
```

**Reply by junnplus** [reply]
I'm sorry for the late reply. What's your expected? Do you want to specify a platform to commit the image?

**Reply by huanghantao** [reply]
yes, specify the platform to `linux/amd64` when commit the image, because I don't want this warning.

**Comment by fahedouch** [comment, marked as answer]
You are commting a container,  If this container is previously running without platform labels then the commit can't find this label and will set default platform label ( which is the platform that you are running on) . So to avoid this message try to create/run you container with `--platform` flag

**Reply by huanghantao** [reply]
thank you!
